tests/test-cases/Group1-Docker-Commands/1-01-Docker-Info.robot

```diff
@@ -69,21 +86,39 @@ Correct container count
     Should Contain  ${output}  Containers: 1
     Should Contain  ${output}  Running: 1
 
-Check modified resource pool CPU and memory values
+Check modified resource pool CPU and memory limits
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} info
     Should Be Equal As Integers  ${rc}  0
 
-    ${oldcpuval}  ${oldmemval}=  Get resource pool CPU and mem values  ${output}
+    ${oldcpuval}  ${oldmemval}=  Get resource pool CPU and mem limits  ${output}
 
     ${newcpuval}=  Evaluate  ${oldcpuval} - 1
     ${newmemval}=  Evaluate  1000
-    Set resource pool CPU and mem values  ${newcpuval}  ${newmemval}
+    Set resource pool CPU and mem limits  ${newcpuval}  ${newmemval}
 
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} info
     Should Be Equal As Integers  ${rc}  0
 
-    ${cpuval}  ${memval}=  Get resource pool CPU and mem values  ${output}
+    ${cpuval}  ${memval}=  Get resource pool CPU and mem limits  ${output}
     Should Be Equal As Integers  ${cpuval}  ${newcpuval}
     Should Be Equal As Integers  ${memval}  ${newmemval}
 
-    Set resource pool CPU and mem values  ${oldcpuval}  ${oldmemval}
+    Set resource pool CPU and mem limits  ${oldcpuval}  ${oldmemval}
+
+Check updated resource pool CPU and memory usages
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} info
+    Should Be Equal As Integers  ${rc}  0
+
+    ${oldcpuval}  ${oldmemval}=  Get resource pool CPU and mem usages  ${output}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d busybox /bin/top
+    Should Be Equal As Integers  ${rc}  0
+
+    Sleep  10s  wait for vsphere stats update
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} info
+    Should Be Equal As Integers  ${rc}  0
+
+    ${cpuval}  ${memval}=  Get resource pool CPU and mem usages  ${output}
+    Should Not Be Equal As Integers  ${oldcpuval}  ${cpuval}
```

Comment:
Should we enforce a stricter comparison for these using less than / greater than instead of not equal?